src_c/rect.c

```diff
@@ -956,25 +956,36 @@ pg_rect_clipline(pgRectObject *self, PyObject *args)
     return Py_BuildValue("((ii)(ii))", x1, y1, x2, y2);
 }
 
-static PyObject *
-pg_rect_contains(pgRectObject *self, PyObject *args)
+static int
+_pg_rect_contains(pgRectObject *self, PyObject *arg)
 {
     int contained;
-    GAME_Rect *argrect, temp;
-
-    if (!(argrect = pgRect_FromObject(args, &temp))) {
-        return RAISE(PyExc_TypeError, "Argument must be rect style object");
+    GAME_Rect *argrect, temp_arg;
+    if (!(argrect = pgRect_FromObject((PyObject *)arg, &temp_arg))) {
+        return -1;
     }
 
     contained = (self->r.x <= argrect->x) && (self->r.y <= argrect->y) &&
                 (self->r.x + self->r.w >= argrect->x + argrect->w) &&
                 (self->r.y + self->r.h >= argrect->y + argrect->h) &&
                 (self->r.x + self->r.w > argrect->x) &&
                 (self->r.y + self->r.h > argrect->y);
+    return contained;
```

Comment:
minor nitpick here, you can probably return `contained` directly without storing it in a temporary variable :)

Reply:
Done!